### PR TITLE
Add Application Page

### LINKED ISSUE #6: Add Application Page
Currently candidates apply through a Google Form that is made each semester. This should become a page within Inquisitor. It will essentially be the same form, but now we can save candidate information to our database right when they apply, making it easy to auto fill information when conducting interviews.

### PR DESCRIPTION
**Pull request by vitorvicente**
Application Form Page (resolves #6)

- Form with candidate data
  - Name
  - Email
  - Resume
  - Class Year
  - 112 Status
  - 330 Status
  - Major
  - Two questions based on last semester's app
  - Saturday/Sunday Time Slot availability
- POST request to future API
- Re-direction to dummy page that thanks them for applying and mentions that they will get an email after doing so.

Everything seems to work besides the POST request (which is untested) and the resume file system.

### CODE REVIEW

**Comment by ROODAY** [comment]
@Warren-Partridge feel free to review this, I will as well, but don't merge this in just yet. Let's wait for the database to be ready so we can have this working end to end before merging. @vitorvicente feel free to pick up another task in the meantime if you're up for it.

File: src/components/Application.js [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+      <Container className="application-form-wrapper">
+        <Row>
+          <Col className="col-md-12 text-center">
+            <img src={logo} alt="UPE Logo" height="256" width="256" />
+            <h1>Application Submitted!</h1>
+          </Col>
+        </Row>
+        <Row className="thanks-text">
+          <Col className="col-md-12 text-center">
+            <p>Thank you for applying to join UPE, we will be getting back to you by Friday the XX with a timeslot for your interview. </p>
```
Comment (Warren-Partridge) [inline src/components/Application.js]: Nit: remove comma splice
```suggestion
            <p>Thank you for applying to join UPE. We will be getting back to you by Friday the XX with a timeslot for your interview. </p>
```

File: src/styles/main.css [hunk 2]
```diff
@@ -67,6 +67,64 @@ p.error-msg {
...
+.application-form-wrapper p {
+  font-size: 12px;
+}
+
+.application-form-wrapper .thanks-text p {
+  font-size: 20px;
+}
+
+.application-form-wrapper .row {
+	padding-top: 50px;
```
Comment (Warren-Partridge) [inline src/styles/main.css]: Nit: 2 spaces instead of tabs

```suggestion
  padding-top: 50px;
```

File: src/styles/main.css [hunk 3]
```diff
@@ -67,6 +67,64 @@ p.error-msg {
...
+}
+
+.application-form-wrapper .row-button {
+	padding-top: 50px;
+	padding-bottom: 50px;
+	text-align: center;
```
Comment (Warren-Partridge) [inline src/styles/main.css]: Nit: more indentation

```suggestion
  padding-top: 50px;
  padding-bottom: 50px;
  text-align: center;
```

File: src/components/Application.js [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+		        </Col>
+		      </Row>
+		      <Row>
+		        <Col>
+			        <h5>When are you available for interviews?</h5>
+			      </Col>
+		      </Row>
+		      <Row>
+			      <Col>
+			        <h6>Saturday - 01/01/20</h6>
```
Comment (Warren-Partridge) [inline src/components/Application.js]: It might be neat to dynamically set dates in the future too.

I really like how `timeslots` works. Nice.

Comment (ROODAY) [reply src/components/Application.js]: Yeah I agree, once the database is setup we should make as many parts of the application as we can to be configurable.

File: src/config/Application.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+const AppQuestions = {
```
Comment (ROODAY) [inline src/config/Application.js]: Not sure if this file makes sense in the application, as moving forward we're going to keep things like this inside of the database, configurable by an admin interface. The config folder should be configuration settings that affect the running of the application instance (currently it doesn't follow this rule with how interview questions are stored there, but that will change once the database is ready).

**Comment by vitorvicente** [comment]
My bad didn't mean to delete this

**Comment by ROODAY** [comment]
Merged into the gatsby branch with 9b7bba58d8736c54303e737662e2f98c8d1c34f9

**Pull request closed without merging**